packages/feedback/src/modal/components/Dialog.css.ts

```diff
@@ -237,19 +238,64 @@ const SUCCESS = `
 }
 `;
 
+function getThemedCssVariables(theme: FeedbackInternalOptions['themeLight']): string {
+  return `
+  --submit-background: ${theme.submitBackground};
+  --submit-background-hover: ${theme.submitBackgroundHover};
+  --submit-border: ${theme.submitBorder};
+  --submit-outline-focus: ${theme.submitOutlineFocus};
+  --submit-foreground: ${theme.submitForeground};
+  --submit-foreground-hover: ${theme.submitForegroundHover};
+
+  --cancel-background: ${theme.cancelBackground};
+  --cancel-background-hover: ${theme.cancelBackgroundHover};
+  --cancel-border: ${theme.cancelBorder};
+  --cancel-outline-focus: ${theme.cancelOutlineFocus};
+  --cancel-foreground: ${theme.cancelForeground};
+  --cancel-foreground-hover: ${theme.cancelForegroundHover};
```

Comment:
should prob rename these from submit to button since we now have the screenshot button too